### ISSUE TITLE
Slice bounds out of range from decode function

### ISSUE THREAD

**Issue by saiKokkonda** [sv/kdbgo#29]
Hi, we are trying to make multiple select queries in parallel to retrieve data from kdb. 

query:-  select open:first price, high:max price, low:min price, close:last price, volume:sum volume by sym from trade where time within time1 time 2 

we are encountering an error : "Slice bounds out of range" from Decode function. 

panic: slice bounds out of range
goroutine 106 [running]:
bufio.(*Reader).Read(0xc000049380, 0xc00112d328, 0x8, 0x8, 0xc00f669c08, 0x441cd3, 0x8)
	C:/Go/src/bufio/bufio.go:234 +0x3ce
io.ReadAtLeast(0xe339e0, 0xc000049380, 0xc00112d328, 0x8, 0x8, 0x8, 0x5, 0xc000dc7560, 0x0)
	C:/Go/src/io/io.go:310 +0x8f
io.ReadFull(...)
	C:/Go/src/io/io.go:329
encoding/binary.Read(0xe339e0, 0xc000049380, 0xe4a140, 0x1487ec0, 0xbd22c0, 0xc00112d2f8, 0x0, 0xc00f669e28)
github.com/sv/kdbgo.Decode(0xc000049380, 0xc00011c270, 0xc00011c201, 0xc000621120, 0x0)

This error occurs only when there is huge data for processing. 
We don't see this error when there is only one select query.
Could some one please look into this ? and recommend a fix.

Environment
kdb windows 32bit
go version go1.12.7 windows/amd64
OS: windows

**Comment by saiKokkonda** [comment]
i am closing this issue, since it is a duplicate of https://github.com/sv/kdbgo/issues/5 . I have created individual connection for each go routine and its fine now.
